### DISCUSSION TITLE
what is the scanner field strength in demo :)

### DISCUSSION THREAD

**Discussion by MqiJiang** [JuliaHealth/KomaMRI.jl#500] [Q&A] [answered]
hi I am trying to create my pulse sequence and simulating it using KomaMRI.
For reference i wonder what's the scanner field strength in the codes in examples\1.sequences\Pulseq_create_seq 🧐

thank you
![图片](https://github.com/user-attachments/assets/37bb3150-a614-4180-94fe-f8dc06b0e764)

**Comment by cncastillo** [comment, marked as answer]
Hi! For the moment, the field strength does not affect the simulation; you need to change properties like T1, T2, off-resonance, etc, by hand depending on the field strength of interest.

Let me know if you need any help :)!
